contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -165,6 +176,7 @@ contract TrueCreditAgency is UpgradeableClaimable {
         uint256 bitMap = usedBucketsBitmap;
         CreditScoreBucket[256] storage creditScoreBuckets = buckets[pool];
         uint256 timeNow = block.timestamp;
+        uint256 partialRate = utilizationAdjustmentRate(pool).add(riskPremium);
```

Comment:
If we decide to keep this (pending this comment: https://github.com/trusttoken/smart-contracts/pull/779/files#r675697168), swap the order to `riskPremium + utilizationAdjustmentRate` to match `currentRate()`'s order. I think it also reads more intuitively left-to-right.